executor.go

```diff
@@ -1675,7 +1674,21 @@ func (e *executor) translateCall(index string, idx *Index, c *pql.Call) error {
 			// will raise an error downstream when it's used.
 			return nil
 		}
-		if field.keys() {
+
+		// Bool field keys do not use the translater because there
```

Comment:
the linter seems to think that "translater" is spelled "translator". I think both are accepted, but we may as well be consistent and keep the linter happy.

Reply:
fixed. i pushed a new commit, so this will need approval.